src/datasources/cache/redis.cache.service.spec.ts

```diff
@@ -5,6 +5,7 @@ import { RedisCacheService } from '@/datasources/cache/redis.cache.service';
 import { RedisClientType } from 'redis';
 import clearAllMocks = jest.clearAllMocks;
 import { fakeJson } from '@/__tests__/faker';
+import { IConfigurationService } from '@/config/configuration.service.interface';
 
 const redisClientType = {
```

Comment:
Out of scope for this PR but we should probably consider converting this to an integration test with Redis itself as we have integration tests at this level with the Postgres database for example.

Reply:
I agree, it's a good idea for a follow-up task, happy to tackle it separately 👍